packages/website/docs/ActionableItem.md

```diff
@@ -0,0 +1,13 @@
+# Best practices
+Use when the actions are secondary and there is no need to display them on the page itself.
+
+It should always be prompted from a button, from which it is associated.
+
+The actionnables shown must be contextual and suited to the user needs based on its selection.
+
+Use the divider line to divide actions into logical groups.
+
+## Labeling
+- Keep titles short, preferably **under three words**.
+- Write for scannability
+- Write the labels in sentence case and use verbs to indicate that an action will be taken
```

Comment:
```suggestion
Keep labels short, preferably **under three words**. Use trigger words to clearly indicate the action performed by the button. A trigger word is typically a verb that influences users into clicking due to its specificity. For example, use "View profile" rather than "Open profile", and "Create playlist" rather than "Save".

The first word of the label should be a verb. If adding a noun is required for context, avoid including an article. For example, write "Add filter" rather than "Add a filter".
```
@johnnyCoveo I'm reusing and slightly tweaking what we had for the Button, just for coherence. Let me know if we must tweak further.